[2024-09-05T00:34:27.734+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-05T00:34:27.798+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_etl_dag.translate_and_clean_data scheduled__2024-08-02T00:00:00+00:00 [queued]>
[2024-09-05T00:34:27.838+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_etl_dag.translate_and_clean_data scheduled__2024-08-02T00:00:00+00:00 [queued]>
[2024-09-05T00:34:27.907+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 2
[2024-09-05T00:34:28.007+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): translate_and_clean_data> on 2024-08-02 00:00:00+00:00
[2024-09-05T00:34:28.019+0000] {warnings.py:112} WARNING - /usr/local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=292) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-05T00:34:28.032+0000] {standard_task_runner.py:72} INFO - Started process 296 to run task
[2024-09-05T00:34:28.025+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather_etl_dag', 'translate_and_clean_data', 'scheduled__2024-08-02T00:00:00+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/weather_etl_dag.py', '--cfg-path', '/tmp/tmpi5hebqq4']
[2024-09-05T00:34:28.039+0000] {standard_task_runner.py:105} INFO - Job 40: Subtask translate_and_clean_data
[2024-09-05T00:34:28.444+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_etl_dag.translate_and_clean_data scheduled__2024-08-02T00:00:00+00:00 [running]> on host b778581f4b0f
[2024-09-05T00:34:30.124+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_etl_dag' AIRFLOW_CTX_TASK_ID='translate_and_clean_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-02T00:00:00+00:00'
[2024-09-05T00:34:30.145+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-05T00:34:30.257+0000] {logging_mixin.py:190} INFO - Recuperando DataFrame de pronósticos desde XCom
[2024-09-05T00:34:30.465+0000] {logging_mixin.py:190} INFO - Número de registros en pronósticos: 18
[2024-09-05T00:34:30.522+0000] {logging_mixin.py:190} INFO - Pronósticos antes de limpiar y traducir:    cve_mun            municipio  ... wind_kph  humidity
0        1    AMEALCO DE BONFIL  ...     16.9        77
1        2      PINAL DE AMOLES  ...     19.8        77
2        3          ARROYO SECO  ...     19.1        82
3        4  CADEREYTA DE MONTES  ...     16.2        78
4        5                COLON  ...     19.8        77

[5 rows x 10 columns]
[2024-09-05T00:35:33.393+0000] {logging_mixin.py:190} INFO - Pronósticos después de limpiar y traducir:    cve_mun            municipio  ... viento_kph  humedad
0        1    Amealco De Bonfil  ...       16.9       77
1        2      Pinal De Amoles  ...       19.8       77
2        3          Arroyo Seco  ...       19.1       82
3        4  Cadereyta De Montes  ...       16.2       78
4        5                Colon  ...       19.8       77

[5 rows x 10 columns]
[2024-09-05T00:35:33.494+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-05T00:35:33.509+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-05T00:35:33.510+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_etl_dag, task_id=translate_and_clean_data, run_id=scheduled__2024-08-02T00:00:00+00:00, execution_date=20240802T000000, start_date=20240905T003427, end_date=20240905T003533
[2024-09-05T00:35:33.644+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-05T00:35:33.698+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
[2024-09-05T18:07:05.533+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-05T18:07:05.634+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_etl_dag.translate_and_clean_data scheduled__2024-08-02T00:00:00+00:00 [queued]>
[2024-09-05T18:07:05.646+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_etl_dag.translate_and_clean_data scheduled__2024-08-02T00:00:00+00:00 [queued]>
[2024-09-05T18:07:05.648+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 2
[2024-09-05T18:07:05.769+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): translate_and_clean_data> on 2024-08-02 00:00:00+00:00
[2024-09-05T18:07:05.773+0000] {warnings.py:112} WARNING - /usr/local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=444) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-05T18:07:05.775+0000] {standard_task_runner.py:72} INFO - Started process 446 to run task
[2024-09-05T18:07:05.776+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather_etl_dag', 'translate_and_clean_data', 'scheduled__2024-08-02T00:00:00+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/weather_etl_dag.py', '--cfg-path', '/tmp/tmpqus9h697']
[2024-09-05T18:07:05.779+0000] {standard_task_runner.py:105} INFO - Job 29: Subtask translate_and_clean_data
[2024-09-05T18:07:05.932+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_etl_dag.translate_and_clean_data scheduled__2024-08-02T00:00:00+00:00 [running]> on host 0b8eaea5e88c
[2024-09-05T18:07:06.251+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_etl_dag' AIRFLOW_CTX_TASK_ID='translate_and_clean_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-02T00:00:00+00:00'
[2024-09-05T18:07:06.252+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-05T18:07:06.268+0000] {logging_mixin.py:190} INFO - Recuperando DataFrame de pronósticos desde XCom
[2024-09-05T18:07:06.306+0000] {logging_mixin.py:190} INFO - Número de registros en pronósticos: 18
[2024-09-05T18:07:06.320+0000] {logging_mixin.py:190} INFO - Pronósticos antes de limpiar y traducir:    cve_mun            municipio  ... wind_kph  humidity
0        1    AMEALCO DE BONFIL  ...     18.0        78
1        2      PINAL DE AMOLES  ...     16.6        75
2        3          ARROYO SECO  ...     17.3        84
3        4  CADEREYTA DE MONTES  ...     14.0        79
4        5                COLON  ...     16.6        75

[5 rows x 10 columns]
[2024-09-05T18:08:06.397+0000] {logging_mixin.py:190} INFO - Pronósticos después de limpiar y traducir:    cve_mun            municipio  ... viento_kph  humedad
0        1    Amealco De Bonfil  ...       18.0       78
1        2      Pinal De Amoles  ...       16.6       75
2        3          Arroyo Seco  ...       17.3       84
3        4  Cadereyta De Montes  ...       14.0       79
4        5                Colon  ...       16.6       75

[5 rows x 10 columns]
[2024-09-05T18:08:06.460+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-05T18:08:06.472+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-05T18:08:06.473+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_etl_dag, task_id=translate_and_clean_data, run_id=scheduled__2024-08-02T00:00:00+00:00, execution_date=20240802T000000, start_date=20240905T180705, end_date=20240905T180806
[2024-09-05T18:08:06.551+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-05T18:08:06.623+0000] {taskinstance.py:3891} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-05T18:08:06.672+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
